node_modules/gh-rest/lib/rest/user.js

```diff
@@ -98,7 +99,8 @@ module.exports = function(client) {
             'emailPreference': opts.emailPreference,
             'isAdmin': opts.isAdmin,
             'recaptchaChallenge': opts.recaptchaChallenge,
-            'recaptchaResponse': opts.recaptchaResponse
+            'recaptchaResponse': opts.recaptchaResponse,
+            'termsAndConditions': opts.termsAndConditions
```

Comment:
Should this be a date of the last time the user has accepted terms and conditions? That way if they change, we can determine that users need to re-accept them.